spec/factories/books.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :book do
+    title "MyString"
```

Comment:
Use [Faker](https://github.com/stympy/faker) for this, like Faker::Lorem.sentence